examples/apps/attributable-map/package.json

```diff
@@ -0,0 +1,97 @@
+{
+	"name": "@fluid-example/attributable-map",
+	"version": "2.0.0-internal.4.3.0",
+	"private": true,
+	"description": "Minimal Fluid Container & Data Object sample to implement a collaborative dice roller as a standalone app.",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "examples/apps/attributable-map"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"main": "lib/index.js",
+	"module": "lib/index.js",
+	"types": "lib/index.d.ts",
+	"scripts": {
+		"build": "concurrently npm:build:compile npm:lint",
+		"build:compile": "npm run build:esnext",
+		"build:esnext": "tsc",
+		"build:full": "concurrently npm:build npm:webpack",
+		"build:full:compile": "concurrently npm:build:compile npm:webpack",
+		"clean": "rimraf dist lib *.tsbuildinfo *.build.log",
+		"eslint": "eslint --format stylish src",
+		"eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
+		"format": "npm run prettier:fix",
+		"lint": "npm run prettier && npm run eslint",
+		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
+		"prepack": "npm run webpack",
+		"prettier": "prettier --check . --ignore-path ../../../.prettierignore",
+		"prettier:fix": "prettier --write . --ignore-path ../../../.prettierignore",
+		"start": "webpack serve",
+		"webpack": "webpack --env production",
+		"webpack:dev": "webpack --env development"
+	},
+	"dependencies": {
+		"@fluid-example/example-utils": "workspace:~",
+		"@fluid-experimental/attributable-map": "workspace:~",
+		"@fluid-experimental/attributor": "workspace:~",
+		"@fluidframework/aqueduct": "workspace:~",
+		"@fluidframework/common-definitions": "^0.20.1",
+		"@fluidframework/container-definitions": "workspace:~",
+		"@fluidframework/container-runtime-definitions": "workspace:~",
+		"@fluidframework/core-interfaces": "workspace:~",
+		"@fluidframework/runtime-utils": "workspace:~",
+		"css-loader": "^1.0.0",
```

Comment:
are css-loader and style-loader necessary? doesn't look like you use either in your webpack config.